src/adhocracy_core/adhocracy_core/interfaces.py

```diff
@@ -249,6 +249,13 @@ def delete_field_values(fields: [str]):
         """Delete values for every field name in `fields`."""
 
 
+class ISheetRequirePassword(ISheet):
+    """Sheet Interface indicating that a password if required for editing.
+
+    To edit such a Sheet the password needs to be send in the request.
```

Comment:
This could be more specific, e.g. "… the password needs to be sind in the request header.". I also found this cound this confusing in some other docstrings.